pymc3/gp/gp.py

```diff
@@ -411,10 +410,6 @@ def sample_gp(trace, gp, X_new, n_samples=None, obs_noise=True,
     obs_noise : bool
         Flag for including observation noise in sample.  Does not apply to GPs
         used with non-conjugate likelihoods.  Defaults to False.
-    cov_func : Covariance
```

Comment:
Ah, the arguments should also be gone.  This is where I ran into problems trying to support additive gps.  I'm submitting a new pr with pretty big refactor soon...